src/routes/docs/products/ai/integrations/replicate/+page.markdoc

```diff
@@ -3,6 +3,369 @@ layout: article
+  <div class="card u-flex u-gap-24 u-flex-vertical">
+    <div class="u-flex u-cross-center u-gap-8">
+      <div class="input-text-wrapper is-with-end-button u-width-full-line">
+        <input x-model="prompt" type="search" placeholder="Prompt" />
+        <div class="icon-search" aria-hidden="true"></div>
+      </div>
+      <div class="select u-width-140">
+        <select x-model="type">
+          <option value="text">Text</option>
+          <option value="image">Image</option>
+          <option value="audio">Audio</option>
+        </select>
+        <span class="icon-cheveron-down" aria-hidden="true"></span>
+      </div>
+
+      <button class="button" x-bind:disabled="loading"
+        x-on:click="async () => { loading = true; answer = {type: '', answer: ''}; try { answer = await onSubmit(prompt, type) } catch(err) { console.error(err); } finally { loading = false; } }">
+        <span class="text">Generate</span>
+      </button>
+    </div>
+    <template x-if="answer.type">
+      <div class="u-flex u-flex-vertical u-gap-12">
+        <div class="u-flex u-flex-vertical u-gap-12 card">
+          <div class="u-flex u-gap-12">
+            <h5 class="eyebrow-heading-2">Result:</h5>
+          </div>
+          <template x-if="answer.type === 'image'" class="u-flex u-gap-12">
+            <img class="u-max-width-400" x-bind:src="answer.response" alt="Replicate output" />
+          </template>
+          <template x-if="answer.type === 'audio'" class="u-flex u-gap-12">
+            <audio x-bind:src="answer.response" controls></audio>
+          </template>
+          <template x-if="answer.type === 'text'" class="u-flex u-gap-12">
+            <p class="u-color-text-gray" x-text="answer.response"></p>
+          </template>
+        </div>
+      </div>
+    </template>
+  </div>
+</div>
+```
+
+This code might seem quite complex if you've never used AlpineJS so we'll break it down a bit:
+
+1. `x-data="{ prompt: '', answer: '', loading: false }"` defines our state for the application.
+2. `x-model="prompt"` and `x-model="type"` is a two-way binding that will bind the value of the input and select to the `prompt` and `type` variables which will use later to post our form.
+3. `x-bind:disabled="loading"` will disable the button if the `loading` state variable is true.
+4. The `x-on:click` binds an event listener that will call the `onSubmit` function we defined earlier within `<script>` when the button is clicked. This function will set the `loading` state to true, clear the `answer` variable, and then call the `onSubmit` function. If the function fails, it will log the error to the console and set the `loading` state back to false.
+5. `x-if="answer"` will only render the completion if the `answer` variable is not empty.
+6. We use multiple `x-if` conditions to render the completion based on the type of completion we receive from the API using `x-text` and `x-bind` respectively.
```

Comment:
Irrelevant, write something similar to what I suggested in the Perplexity comment here.